roles/openshift_manageiq/vars/main.yml

```diff
@@ -15,6 +15,12 @@ manageiq_service_account:
     metadata:
       name: management-admin
 
+manageiq_image_inspector_service_account:
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: miq-inspector
```

Comment:
Again, not asking for changes, just gathering general feedback for future improvements. @abutcher @sdodson thoughts on having a filter that builds up the service_account template instead of templating it out all the time?